Portal/src/Datahub.Infrastructure/ConfigureServices.cs

```diff
@@ -36,6 +45,34 @@ public static class ConfigureServices
         services.AddScoped<IUsersStatusService,UsersStatusService>();
         services.AddSingleton<IDatahubCatalogSearch, DatahubCatalogSearch>();
 
+        services.AddMassTransit(x =>
+        {
+            x.AddConsumer<EmailNotificationConsumer>();
+            if (whereAmI == "Development")
+            {
+                x.UsingInMemory((context, cfg) =>
+                {
+                    cfg.ConfigureEndpoints(context);
+                    cfg.UseConsumeFilter(typeof(LoggingFilter<>), context);
+                });
+            }
+            else
+            {
+                x.UsingAzureServiceBus((context, cfg) =>
+                {
+                    var serviceBusConnectingString = configuration["DatahubServiceBusConnectionString"]
+                        ?? configuration["DatahubServiceBus:ConnectionString"];
+
+                    cfg.Host(serviceBusConnectingString);
+                    cfg.ReceiveEndpoint("email-notification", e =>
+                    {
+                        e.ConfigureConsumer<EmailNotificationConsumer>(context);
+                    });
+                    cfg.UseConsumeFilter(typeof(LoggingFilter<>), context);
+                });
+            }
+        });
```

Comment:
I tried to create an extension but passing consumer seems to be non-trivial. Leaving it 'as is' for now

Reply:
@vb-ssc The portal doesn't consume emails. Why is this in this function?

Reply:
Portal has Email page where user can send email to everyone or to specific user. This is existing functionality. Current ticket just changes the way how email is handled (instead of Azure Queue via MediatR it uses Azure Service Bus via MassTransit)

Reply:
Portal pushes messages to the queue to send them. This adds a receiver for the queue

Reply:
Existing code resides in Datahub.Functions\EmailNotificationHandler.cs
```
[Function("EmailNotificationHandler")]
public async Task Run([QueueTrigger("%QueueEmailNotification%", Connection = "DatahubStorageConnectionString")] string requestMessage)
```
Are you suggesting to move receiver to Datahub.Functions ?